src/main/scala/scalaz/parsers/transform.scala

```diff
@@ -98,30 +120,28 @@ object Transform {
 
   trait DeriveTransformFunctions[=>:[_, _]] extends TransformClass[=>:] {
 
+    implicit val self: CategoryClass[=>:] = this
```

Comment:
Shouldn't it be private everywhere?
Otherwise it seems to be overridden and can also be imported via wildcard import.

Reply:
agreed.  fixing then I think we are good to merge.

Reply:
Thanks!